tests/foreman/ui/test_discoveredhost.py

```diff
@@ -141,6 +135,234 @@ def setUpClass(cls):
         cls.discovery_auto.value = 'True'
         cls.discovery_auto.update({'value'})
 
+        # Create a new Life-Cycle environment
```

Comment:
All things that are happening there seems very similar to:
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_host.py#L92

We need to have common helper method to not have such big duplication of code. Maybe, something like in:
https://github.com/SatelliteQE/robottelo/blob/master/robottelo/cli/factory.py#L2720